docs/proposals/n000-proposals.md

```diff
@@ -0,0 +1,92 @@
+---
+np: 000
+title: Proposals
+status: Draft
+type: Meta
+author: George Antoniadis <george@noodles.gr>
+created: 2020-12-20
+---
+
+# np000 - Proposals
+
+## Simple Summary
+
+A formal approach to proposing new features or updates for nimona.
+
+## Problem Statement
+
+Until now nimona's spec and protocol was very fluid in order to allow us to
+quickly iterate without fear or braking things. This has resulted in a lack of
+specifications and concrete documentation.
+
+As we are moving closer to a more solid protocol we'll need to spend more time
+designing updates in order to make sure everything is backwards compatible and
+does not break apps.
+
+In addition to this, we need a good way to help newcomers understand the various
+parts of nimona and the rationale behind some of the design decisions.
+
+## Proposal
+
+Instead of trying to maintain specification docs, create distinct and standalone
+proposals for each new feature or update.
+
+Documentation will serve as a TL;DR and attempt to guide the reader through the
+various proposals.
+
+The proposal process itself is influenced by
+[golang's](https://github.com/golang/proposal#design-documents) and
+[ethereum's](https://eips.ethereum.org/EIPS/eip-1)
+proposal processes.
+
+### Index example
+
+* Objects
+  * np001 - Hinted objects
```

Comment:
We could have another level of numbers to link various proposals together. ie `np001-01 - Hinted Objects v2`

Reply:
Hm, interesting point, I think we can deal with this as we move forward. I'll look into how RFCs and other people deal with this for now.